404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
```

Comment:
@luke-jr That kind of wording sounds reasonable.

Similarly, saying things like "payments in 10 minutes" is pretty close to the truth already - "irreversible payments in under an hour" for instance would be quite accurate.